.scripts/mocha_runner.js

```diff
@@ -11,6 +11,8 @@ require('babel-polyfill');
 var jsdom = require('jsdom').jsdom;
 
 var exposedProperties = ['window', 'navigator', 'document'];
+var fileExts = ['jpg', 'png', 'gif', 'eot', 'svg', 'ttf', 'woff', 'woff2'];
```

Comment:
I think this is because we included css.
So, just add a single entry for just handling `.css`